radix-engine/src/model/substates.rs

```diff
@@ -12,6 +12,9 @@ use radix_engine_interface::data::IndexedScryptoValue;
 pub enum PersistedSubstate {
     Global(GlobalAddressSubstate),
     EpochManager(EpochManagerSubstate),
+    CurrentTime(CurrentTimeSubstate),
+    CurrentTimeRoundedToSeconds(CurrentTimeRoundedToSecondsSubstate),
+    CurrentTimeRoundedToMinutes(CurrentTimeRoundedToMinutesSubstate),
```

Comment:
One option I think we may want to think about is whether we even want to persist time. It might be better to just be something that is inputted at the beginning of execution.